compile.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo -e "\n\n\tMicrosoft RP2040 LampArray Sample run\n\n"
+
+
+echo -e "\n\tStep 2.\n"
+sudo git clone https://github.com/microsoft/RP2040MacropadHidSample.git
+
+echo -e "\n\tStep 3.\n"
+sudo apt install build-essential
+
+echo -e "\n\tStep 4\n"
+sudo apt install cmake gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
+sudo git clone https://github.com/raspberrypi/pico-sdk.git
+sudo rm ./RP2040MacropadHidSample/src/pico_sdk_import.cmake
+sudo cp ./pico-sdk/external/pico_sdk_import.cmake ./RP2040MacropadHidSample/src/pico_sdk_import.cmake
+export PICO_SDK_PATH=$PWD/pico-sdk
+
+echo -e "\n\tStep 5.\n"
+cd ./pico-sdk
+sudo git submodule update --init
+cd ..
+
+echo -e "\n\tStep 6.\n"
+sudo git clone https://github.com/rsolorzanomsft/tinyusb.git
```

Comment:
This step has been updated, now you just need to update the tinyusb branch to master

Reply:
I'm talking about step 6 here.